src/modules/rewards/components/LeaderBoardListItem.tsx

```diff
@@ -10,10 +10,13 @@ import { useBlocksTheme } from 'blocks/Blocks.hooks';
 //Components
 import { Box, Text } from 'blocks';
 
+//Utility helper
+import { shortenText } from 'helpers/UtilityHelper';
+
```

Comment:
We also need to show chain details on hover I see that is removed in this PR. Any reason? https://github.com/push-protocol/push-dapp/pull/1653/files#diff-9b00b69b1487ed32a8c5af5d92c37b7efa50cdf378c5a671b9463413e4bdd897